File: clang/test/Driver/flang/multiple-inputs.f90
! Check that flang driver can handle multiple inputs at once.

! RUN: %clang --driver-mode=flang -### -fsyntax-only %S/Inputs/one.f90 %S/Inputs/two.f90 2>&1 | FileCheck --check-prefixes=CHECK-SYNTAX-ONLY %s
! CHECK-SYNTAX-ONLY-LABEL: "{{[^"]*}}flang" "-fc1"
! CHECK-SYNTAX-ONLY: "{{[^"]*}}/Inputs/one.f90"
! CHECK-SYNTAX-ONLY-LABEL: "{{[^"]*}}flang" "-fc1"
! CHECK-SYNTAX-ONLY: "{{[^"]*}}/Inputs/two.f90"

File: clang/test/Format/multiple-inputs.cpp
// RUN: cp %s %t-1.cpp
// RUN: cp %s %t-2.cpp
// RUN: clang-format -style=LLVM %t-1.cpp %t-2.cpp|FileCheck -strict-whitespace %s

// CHECK: {{^int\ \*i;}}
// CHECK: {{^int\ \*i;}}
 int   *  i  ;
